.gitignore

```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
+bin/
+obj/
+.vscode/
+.vs/
+app.db
+
+=======
```

Comment:
```suggestion
```
There seems to be the remnants of a merge conflict in the `.gitignore` file, remove these artifacts